app/assets/stylesheets/pages/home.scss

```diff
@@ -73,18 +112,47 @@ $small-screen: em(420);
 
 
 #home-title {
-  padding-top: 50px;
-
+  padding-top: 75px;
   @include media(max-width $small-screen) {
-    padding-top: 30px;
+    padding-top: 32px;
   }
 }
 
+#home-subtitle {
+  padding-top: 5px;
+
+  @include media(max-width $medium-screen) {
+    padding-top: 25px;
+  }
+}
+
+.bold-blue {
+  font-weight: bold;
+  color: #477DCA;
+}
+
 #our-projects {
-  text-align: center;
+  position: relative;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered left, position, text-align, top